README.md

```diff
@@ -1,2 +1,55 @@
 # fdk-model-publisher
 Transforms models included in dataservice-descriptions to modelldcat-ap-no compliant models and expose the collection
+
+## Develop and run locally
+### Requirements
+- [pyenv](https://github.com/pyenv/pyenv) (recommended)
+- [poetry](https://python-poetry.org/)
+- [nox](https://nox.thea.codes/en/stable/)
+- [nox-poetry](https://pypi.org/project/nox-poetry/)
```

Comment:
Kan vi kaste bort poetry og bruke pipenv?

Reply:
nei

Reply:
Et mer seriøst svar:

I det hele ser jeg ikke veldig stor forskjell på poetry og pipenv, begge to er legitime valg. Men jeg har problem med pipenv lokalt og da stiller poetry sterkere hos meg :P

Derfor har jeg ikke så stor erfaring med pipenv, men har jo lest litt. Rask sammenligning: Bedre støtte for eldre python versjoner i pipenv, men vi trenger ikke støtte eldre versjoner. Enkelt å switche mellom forskjellige python versjoner i poetry, en veldig grei funksjon, men betyr strengt tatt ikke så mye. Publisering til pypi er uproblematisk med poetry, får inntrykk av at det ikke er sant for pipenv. Akkurat det kan være et stort pluss til poetry, men bare hvis vi skal publisere til pypi. Det er ikke støtte for 'scripts' i poetry, løser det med nox.

Etter min erfaring så er nox et glimrende 'script'-verktøy. Kjører opp et internt miljø for hver enkelt sesjon, som fører til veldig like kjøringer lokalt og ved deploy. Synes strengt tatt at vi også burde bruke det i pipenv-prosjekt...

Så helt ok at din preferanse er pipenv, men jeg ser ikke noe som gjør det til et 'bedre' valg enn poetry.